motor/motor_gridfs.py

```diff
@@ -227,6 +227,7 @@ class AgnosticGridIn(object):
     write = AsyncCommand().unwrap("MotorGridOut")
     writeable = DelegateMethod()
     writelines = AsyncCommand().unwrap("MotorGridOut")
+    _chunk_number = ReadOnlyProperty("_chunk_number")
```

Comment:
We should add `_chunk_number` to synchro's GridIn object, not here, because it's only relevant for a synchro test.